spec/scopes-and-launch-context/index.md

```diff
@@ -172,6 +172,7 @@ Launch context parameter | Example value | Meaning
 ------|---------|-------------------
 `patient` | `"123"`| String value with a patient id, indicating that the app was launched in the context of FHIR Patient 123. If the app has any patient-level scopes, they will be scoped to Patient 123.
 `encounter` | `"123"`| String value with an encounter id, indicating that the app was launched in the context of FHIR Encounter 123.
+`appointment` | `"789"` | String value with an appointment id, indicating that the app was launched in the context of FHIR Appointment 789.
```

Comment:
Interesting. Can you comment on the difference between an appointment and an "Encounter" where status = "planned".

Reply:
I think that's out of the scope of this PR. He's expanding optional support for a `launch/<resourceType>`. Both Appointment and Encounter are valid resources, so he's just making the optional support explicit.

Whether there _should_ be an Appointment resource is another question. :shrug: